src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: block;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.bar_b::placeholder {
+  color: #3d4e61;
+}
+
+.bar_b {
+  outline: none;
+  display: block;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  box-sizing: border-box;
+  line-height: 22px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 26px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-color: #fff;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px;
+}
+
+.bar_b:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
+}
+
+.bar_b:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 900;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+}
+
+.bar_s {
+  font-family: "Avenir", sans-serif;
+  margin-top: 20px ;
+  width: 100%;
+  padding-left: 33px;
+  height: 42px;
+  display: block;
+  box-sizing: border-box;
+  line-height: 19px;
+  font-weight: 400;
+  font-size: 14px;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  background-size: 11px;
+  background-position: 12px 14px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
```

Comment:
you have a lot of duplicated css properties from bar_b class here. Try to avoid it by following advices from checklist. Create just search class and add search--small class which will modify only needed properties for smaller input 
![Screenshot 2023-03-21 at 17 42 04](https://user-images.githubusercontent.com/6526149/229835136-660e57d7-0dc7-4ccc-a47a-cb1fab6e1b75.png)

Reply:
not fixed